website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -169,6 +170,31 @@ builder.
 -   `skip_region_validation` (boolean) - Set to true if you want to skip 
     validation of the region configuration option.  Defaults to false.
 
+-   `source_ami_filter` (object) - Filters used to populate the `source_ami` field.
```

Comment:
I'm open to better ways to document this. This was the best means that occurred to me at the moment.